lib/shopify_api/auth/jwt_payload.rb

```diff
@@ -73,8 +73,7 @@ def ==(other)
 
       sig { params(token: String, api_secret_key: String).returns(T::Hash[String, T.untyped]) }
       def decode_token(token, api_secret_key)
-        JWT.decode(token, api_secret_key, true,
-          { exp_leeway: JWT_EXPIRATION_LEEWAY, algorithm: "HS256" })[0]
+        JWT.decode(token, api_secret_key, true, leeway: JWT_EXPIRATION_LEEWAY, algorithm: "HS256")[0]
```

Comment:
I would prefer to rename the constant to `JWT_LEEWAY` since it doesn't only apply to expiration anymore, but it's a public constant and might make this a breaking change, so I decided to keep it.

Reply:
Oh yeah good call, I'll do that then 👍